cupy-feedstock/recipe/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set version = "2.5.0" %}
+{% set commit = "2274875a22dfabd55c73c3cfb1d005a8e8000c12"%}
+
+package:
+  name: cupy
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/cupy/cupy.git
+  git_tag: {{ commit }}
+
+build:
+  number: 0
+  skip: True # [not linux64]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+
+  host:
+    # GPU requirements
+    - cudatoolkit {{ cudatoolkit }}*
+    - cudnn {{ cudnn }}*
+    - nccl
+    # other requirements
+    - python
+    - numpy 1.11*
```

Comment:
Does CuPy have a minimal NumPy version, if not you can remove this version pin.  The `conda_build_config.yaml` file and `pin_compatible` will take care of the rest.